### DISCUSSION TITLE
Add support for Structured Data

### DISCUSSION THREAD

**Discussion by ianleeder** [patrickdemooij9/SeoToolkit.Umbraco#124] [Ideas]
One of our SEO requirements is to add "Structured Data" to our new Umbraco site:  
https://developers.google.com/search/docs/appearance/structured-data/intro-structured-data

It looks like it would be a good fit for this package.  At least some of those values would be easy to add (eg [`logo`](https://developers.google.com/search/docs/appearance/structured-data/logo).  Full support with different data types or nested objects may become more complex.

**Comment by patrickdemooij9** [comment]
Yeah, this would be amazing to somehow get working. I've a small prototype ready that I need to spend some more time on. I think the main "issue" is what would be the best way of visualizing/configuring this for the CMS user.
